justfile

```diff
@@ -12,7 +12,9 @@ build:
   @pnpm --filter next-bg-image compile
 
 publish *FLAGS: build
+  @cp README.md next-bg-image/
   @cd next-bg-image && npub {{FLAGS}}
+  @rm next-bg-image/README.md
```

Comment:
during the build, we temporarily copy the readme down into the component dir, so it gets packaged up.